tests/system/python/conftest.py

```diff
@@ -30,18 +31,21 @@
 
 @pytest.fixture
 def clean_setup_fledge_packages(package_build_version):
-    assert os.environ.get('FLEDGE_ROOT') is not None
+    # This  gives the path of directory where fledge is cloned. conftest_file < python < system < tests < ROOT
+    PROJECT_ROOT = Path(__file__).parent.parent.parent.parent
+    SCRIPTS_DIR_ROOT = "{}/tests/system/python/scripts/package/".format(PROJECT_ROOT)
```

Comment:
is this not for non-packages based test? here we have `clean_setup_fledge_packages`

Reply:
No, this is for packages based tests.